includes/definitions/discovery/supermicro-bmc.yaml

```diff
@@ -1,74 +1,169 @@
+                    index: 'fanRPM2.{{ $index }}'
+                    skip_values: 0
                 -
                     oid: sensorTable
                     value: sensorReading
                     num_oid: '.1.3.6.1.4.1.21317.1.3.1.2.{{ $index }}'
                     descr: sensorIDString
-                    index: sensorTableSensorNr.{{ $index }}
-                    low_limit: lcThreshold
-                    low_warn_limit: lncThreshold
-                    warn_limit: uncThreshold
-                    high_limit: ucThreshold
+                    index: 'sensorReading.{{ $index }}'
                     skip_values:
                         -
                             oid: sensorIDString
                             op: 'not_contains'
-                            value: 'FAN'
+                            value: "FAN"
+                        -
+                            oid: lcThreshold
+                            op: '='
+                            value: 0
+                    low_warn_limit: lcThreshold
+                    warn_limit: ucThreshold
+                    low_limit: lnrThreshold
+                    high_limit: unrThreshold
         voltage:
             data:
+                -
+                    oid: inputVoltage
+                    value: inputVoltage
+                    num_oid: '.1.3.6.1.4.1.21317.1.14.2.1.3.{{ $index }}'
+                    descr: 'PSU {{ $index }} Input Voltage'
+                    index: 'inputVoltage.{{ $index }}'
                 -
                     oid: sensorTable
                     value: sensorReading
                     num_oid: '.1.3.6.1.4.1.21317.1.3.1.2.{{ $index }}'
                     descr: sensorIDString
-                    index: sensorTableSensorNr.{{ $index }}
-                    low_limit: lcThreshold
-                    low_warn_limit: lncThreshold
-                    warn_limit: uncThreshold
-                    high_limit: ucThreshold
+                    index: 'sensorReading.{{ $index }}'
                     skip_values:
                         -
                             oid: sensorIDString
-                            op: 'not_in_array'
-                            value:
-                                - Vcpu
-                                - VDIMM
-                                - 12V
-                                - 5VCC
-                                - 3.3VCC
-                                - VBAT
-                                - AVCC
-                                - VSB
+                            op: 'not_contains'
+                            value: "V"
+                        -
+                            oid: sensorIDString
+                            op: 'contains'
+                            value: "Temp"
+                        -
+                            oid: sensorIDString
+                            op: 'in_array'
+                            value: ['VBAT']
```

Comment:
Is VBAT not voltage battery?

Reply:
not on this platform:

https://www.supermicro.com/support/faqs/faq.cfm?faq=31079